pages/web-literacy/MakeLinksFromWebLitSkills.jsx

```diff
@@ -5,12 +5,12 @@ var ReactRouter = require('react-router');
 var Link = ReactRouter.Link;
 var convertToRoute = require("../../lib/util").convertToRoute;
 
-var MakeLinksFromWebLitSkills = function(webLitSkills) {
+var MakeLinksFromWebLitSkills = function(webLitSkills, locale) {
   return webLitSkills.map(function(webLitSkill, index) {
     return (
       <span className="comma-separated-links" key={webLitSkill}>
         <Link
-          to={"/web-literacy/" + convertToRoute(webLitSkillsContent[webLitSkill].topic) + "/" + convertToRoute(webLitSkill) + "/"}
+          to={"/" + locale + "/web-literacy/" + convertToRoute(webLitSkillsContent[webLitSkill].topic) + "/" + convertToRoute(webLitSkill) + "/"}
```

Comment:
This, too, feels like we should make a `<LocalLink>`-like component that takes a regular `to` and then rewrites it internally, putting the locale code in without needing a dev to type it anywhere.